docs/setup-guide/vm_setup.md

```diff
@@ -0,0 +1,137 @@
+#	Virtual Machine Setup and Configuration
+
+## Abstract
+The purpose of this document is to how to setup and configuration Virtual Machine, and describe the each step to create virtual machine, setup the port number.
```

Comment:
same here, "is to how to setup" gramma doesn't seem right

Reply:
also "Virtual Machine" --> "virtual machine"

Reply:
Updated